app/src/main/java/app/wefridge/wefridge/datamodel/ItemController.kt

```diff
@@ -0,0 +1,117 @@
+package app.wefridge.wefridge.datamodel
+
+import android.util.Log
+import app.wefridge.wefridge.exceptions.ItemOwnerMissingException
+import app.wefridge.wefridge.placeholder.PlaceholderContent
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.Timestamp
+import com.google.firebase.firestore.DocumentReference
+import com.google.firebase.firestore.GeoPoint
+import java.lang.Exception
+import kotlin.collections.ArrayList
+
+class ItemController: ItemControllerInterface {
+    private val TAG = "ItemsOnFirebase"
+    private val db = FirebaseFirestore.getInstance()
+    private val items = ArrayList<Item>()
+
+    /*
+    * The function getItems is based on an example provided on
+    * https://firebase.google.com/docs/firestore/query-data/get-data
+    *
+    * The functions deleteItem, overrideItem and addItem are partially
+    * based on code snippets provided by the Firebase Documentation:
+    * https://firebase.google.com/docs/firestore/manage-data/add-data
+    * */
+    override fun getItems(callbackOnSuccess: (ArrayList<Item>) -> kotlin.Unit, callbackOnFailure: (Exception) -> kotlin.Unit) {
+        // TODO: only get items of specifc user and the groups participants
```

Comment:
TODO done?

Reply:
Yes, the first part of this TOD is done.
I would suggest to open a new issue that is about extending the function to deliver items of group participants, too.

Reply:
Sure

Reply:
All right, the issue for that is now listed under #44